includes/EmbedService/Indiana.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare( strict_types=1 );
+
+namespace MediaWiki\Extension\EmbedVideo\EmbedService;
+
+final class Indiana extends AbstractEmbedService {
+
+	/**
+	 * @inheritDoc
+	 */
+	public function getBaseUrl(): string {
+		//return '//player.vimeo.com/video/%1$s';
+		//return '//purl.dlib.indiana.edu/iudl/media/%1$s' . '/embed';
+		return '//media.dlib.indiana.edu/master_files/%1$s' . '/embed';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function getAspectRatio(): ?float {
+		return 600 / 450;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	protected function getDefaultWidth(): int {
+		return 600;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	protected function getDefaultHeight(): int {
+		return 450;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	protected function getUrlRegex(): array {
+		return [
+			'Indiana'
```

Comment:
This part is used to validate the URL of the service. I'm guessing something like the following could work:
`#media\.dlib\.indiana\.edu/master_files/[\d\w\-_]{3,}/embed#i`

To check if this works, you can paste the code into the input called `Regular Expression` at https://regex101.com and the url of a typical embed into the input called  `Test String` 

The test is successful, when the url is highlightet in blue